runtime/fixtures/dart_tool/flutter_build/dart_plugin_registrant.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+void passMessage(String message) native 'PassMessage';
+
+bool didCallRegistrantBeforeEntrypoint = false;
+
+// Test the Dart plugin registrant.
+@pragma('vm:entry-point')
```

Comment:
is there any difference between this file (or test) and https://github.com/flutter/engine/blob/bfb58101c185c1d991f478eb1ff14fb561a0feca/runtime/fixtures/runtime_test.dart#L164-L185?

Reply:
Yep, the path of the file.  The other one can be removed once we land the Framework PR.